recipes/hmmlearn/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "hmmlearn" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "9b087ec7fb217d68b0581b6696a38c0acb54e93f6538ad8e0da6738990f45ca8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/hmmlearn/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like this compiles a C-extension so please include `toolchain` as a `build` requirement. Also see the following about building against numpy:

https://conda-forge.org/docs/meta.html#building-against-numpy